domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -2840,14 +2840,14 @@ public void streamGetExceptionInHandler(TestContext context) throws IOException,
         });
         sr.exceptionHandler(x -> {
           events.append("[exception]");
+          vertx.runOnContext(run -> async.complete());
         });
         sr.endHandler(x -> {
           events.append("[endHandler]");
-          async.complete();
         });
       }));
     async.await(1000);
-    context.assertEquals("[handler][exception][endHandler]", events.toString());
+    context.assertEquals("[handler][exception]", events.toString());
```

Comment:
Good catch. Is async.await(1000) still needed then?

Reply:
Yes, it is still needed, because we need to wait until async.complete() has been called before we can run the assertEquals checks. Otherwise event will be empty.